File: CompSci 1/Unit 4, Operators/Baseball/Baseball.java
public class Baseball {
    public static void main(String[] args) {
        int hits = 27;
        int atBats = 80;
        double battingAvg = 0;
        int roundedAvg = 0;

        battingAvg = (double)hits / atBats;

        battingAvg += 0.0005;

        battingAvg *= 1000;
        roundedAvg = (int)battingAvg;

        System.out.println("Hits = " + hits);
        System.out.println("At Bats = " + atBats);
        System.out.println("-------------------------");
        System.out.println("Batting Average = " + roundedAvg);
    }
}

File: CompSci 1/Unit 4, Operators/Baseball/Baseball.kt
fun main(args: Array<String>) {
    val hits = 27
    val atBats = 80
    var battingAvg = 0.0
    var roundedAvg = 0

    battingAvg = hits.toDouble() / atBats

    battingAvg += 0.0005

    battingAvg *= 1000.0
    roundedAvg = battingAvg.toInt()

    println("Hits = " + hits)
    println("At Bats = " + atBats)
    println("-------------------------")
    println("Batting Average = " + roundedAvg)
}